2021q3/ztop.adoc

```diff
@@ -0,0 +1,25 @@
+== Ztop
+
+Links: +
+link:https://github.com/asomers/ztop[Repository] +
+link:https://www.freshports.org/sysutils/ztop[Port] +
+
+Contact: Alan Somers <asomers@FreeBSD.org> +
+
+`ztop` is a new utility that displays ZFS datasets' I/O in real time, like
+`top`, but for ZFS.  Unlike the existing `zpool iostat`, which only displays
+traffic at the level of a pool, `ztop` displays it at the level of individual
+datasets.
+
+Previously, there was no tool that could display this information.  The
+link:https://github.com/prometheus/node_exporter/pull/1632[Prometheus Node
+Exporter] can export it into Promtheus, from which it can be viewed
+after-the-fact with various other tools, but that requires a large stack of
+software, and isn't real-time.
+
+I started `ztop` this quarter, and it is now fully functional.  However, it has
+revealed a performance problem within the kernel.  In the presence of more than
+a few thousand datasets, the sysctl interface is too slow and `ztop` becomes
+sluggish.  Future work, if I have the time, will address that problem.
+
+Sponsor: Axcient +
```

Comment:
Maybe remove "+" since this is the last line of this section? (Stylistic if nothing else.)